bindings/native/go/go.sum

```diff
@@ -0,0 +1,16 @@
+github.com/awnumar/memcall v0.0.0-20191004114545-73db50fd9f80 h1:8kObYoBO4LNmQ+fLiScBfxEdxF1w2MHlvH/lr9MLaTg=
```

Comment:
Not a go developer, but isn't `go.sum` a generated file? Please do not check in generated sources

Reply:
Its recommended to commit it as well https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file